src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2044,36 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    SendErrorToUi("Unable to find preset file: %s", file_exists.error().message());
+    return;
+  }
+
+#if defined(__linux)
+  const QString program{"dbus-send"};
+  const std::string message_content = absl::StrFormat(
+      "/org/freedesktop/FileManager1 org.freedesktop.FileManager1.ShowItems "
+      "array:string:\"file:///%s\" string:\"\"",
+      preset.file_path().string());
+  QStringList arguments;
+  arguments << "--session"
+            << "--print-reply"
+            << "--dest=org.freedesktop.FileManager1"
+            << "--type=method_call" << QString::fromStdString(message_content);
```

Comment:
Nit: I would prefer to use `QStringList::push_back` because it's similar to `std::vector` instead of the streaming operator.

Reply:
Alterantively the braced initialization syntax also works.

Reply:
Changed to use braced initialisation! Thanks!